ci/Dockerfile.circle_build

```diff
@@ -13,7 +13,8 @@
 #
 # We use OpenJDK 8, Node, and some common browsers from CircleCI's base image
 # see: https://circleci.com/docs/2.0/circleci-images/#language-image-variants
-FROM circleci/openjdk:8-buster-node-browsers
```

Comment:
Specifying the Debian distribution (e.g. `buster`) is no longer supported.  The new images use Ubuntu LTS.

`-node` is now included in `-browsers` (and this particular image installs Node 16)